### DISCUSSION TITLE
HTTP error 500 after updating to version 2.4.28

### DISCUSSION THREAD

**Discussion by bertdynamica** [liedekef/events-made-easy#441] [General]
It happens to me regularly that I get an http error 500 after updating the plugin events-made-easy to a new version. 

I am using the default theme Kadence. When I switch to another theme, for example Spacious, I have no problem.

What can be done about this?

Greetz, Bert

**Comment by liedekef** [comment]
Error 500 indicates a problem with the site, so you'll need to check your logs. It might even be that your security plugin is blocking things ... if it works when switching to another theme, it seems not EME related. But I'll try the Kadence theme (if it is free).
Edit: no issue with the kadence theme, so not reproducable here.

**Comment by bertdynamica** [comment]
Hello Franky,

Thanks for your efforts.

With another web hoster, I have a dummy website that also has EME installed and also Kadence's theme. 

No problems here either. Will check with the web hoster what could be wrong.

Kind regards, Bert
